web/styles/globals.css

```diff
@@ -0,0 +1,40 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@layer base {
+  html {
+    -webkit-tap-highlight-color: transparent;
+  }
+}
+@font-face {
+  font-family: 'icomoon';
+  src: url('../fonts/icomoon.eot?v49ui9');
+  src: url('../fonts/icomoon.eot?v49ui9#iefix') format('embedded-opentype'),
+    url('../fonts/icomoon.ttf?v49ui9') format('truetype'),
+    url('../fonts/icomoon.woff?v49ui9') format('woff'),
+    url('../fonts/icomoon.svg?v49ui9#icomoon') format('svg');
+  font-weight: normal;
+  font-style: normal;
+  font-display: block;
+}
+
+[class^='icon-'],
+[class*=' icon-'] {
```

Comment:
This matches all `icon-` classes. Why do we want to do that?

I'm not actually not sure how this change works overall haha. How can it apply a different font just for the money character? Oh I see you said it's for a character that readex pro doesn't have.